index.html

```diff
@@ -6,24 +6,177 @@
   </head>
   <body>
     <script>
-      alert('Задание №1');
-      var tempC = +prompt('Введите температуру по Цельсию');
-      var TEMP_F = 9 / 5 * tempC + 32;
-      if (isNaN(TEMP_F)) {
-        alert('Нужно ввести число');
+      alert('Задание 1');
+      alert('Такие результаты получаются потому, что при более-менее сложных вычисления ++а и а++ имеют различия');
+    </script>
+    <script>
+      alert('Задание 2');
+      alert('Если преременная а равна 2, то x будет равен 5');
+      var a = 2;
+      var x = 1 + (a *= 2);
+      alert(x);
+    </script>
+    <script>
+      alert('Задание 3');
+      var a = +prompt('Ведите первое целове число');
+      var b = +prompt('Ведите второе целове число');
+      if (isNaN(a) || isNaN(b)) {
+        alert('Одно из введенных значений НЕ число');
+      } else if (!Number.isInteger(a) || !Number.isInteger(b)) {
+        alert('Одно из введенных чисел не целое');
+      } else if (a >= 0 && b >= 0) {
+        var result = a - b;
+        alert('Разность a и b равна ' + result);
+      } else if (a < 0 && b < 0) {
+        var result = a * b;
+        alert('Произведение a и b равна ' + result);
+      } else if ((a < 0 && b >= 0) || (a >= 0 && b < 0)) {
```

Comment:
в проверке этого условия нет смысла
```suggestion
      } else {
```